src/services/passwordRecovery/hooks/index.js

```diff
@@ -49,7 +81,7 @@ exports.after = {
 	all: [],
 	find: [],
 	get: [],
-	create: [],
+	create: [sendInfo],
```

Comment:
Naming sendInfo for send password recover email, maybe it is confused?

Reply:
Why? the service is named passwordRecovery. Sent info is related to password recovery information

Reply:
send email should be part of the service logic and not hidden behind a hook.

Reply:
@tofixx same behaviour like used in teams